src/Servers/Kestrel/Core/src/Internal/Http/HttpHeaders.Generated.cs

```diff
@@ -99,6 +99,106 @@ internal enum KnownHeaderType
         WWWAuthenticate,
     }
 
+    internal partial class HttpHeaders
+    {
+        protected readonly static HashSet<string> s_internedHeaderNames = new HashSet<string>(93, StringComparer.OrdinalIgnoreCase)
```

Comment:
Nit:

```suggestion
        private readonly static HashSet<string> s_internedHeaderNames = new HashSet<string>(93, StringComparer.OrdinalIgnoreCase)
```

Also, `s_` is a runtime convention, but I'm okay with it. It started with the shared code. It's only a matter of time 😆